docs/TOOLS.md

```diff
@@ -1,44 +1,206 @@
+
+![Spyder Python Interpreter Selection](https://raw.githubusercontent.com/exercism/python/main/docs/img/Spyder-Config-Interpreter.png)
+
+Do not forget to click `Apply` once you selected your interpreter.
+
+#### Other features
+
+[Spyder Unittest](https://github.com/spyder-ide/spyder-unittest/releases/latest) - If you want to have a built-in interface for you tests, install this plugin. Clicking the link will bring you to the latest release on GitHub.
+
+[Code analyzer](https://docs.spyder-ide.org/current/panes/pylint.html#using-the-code-analyzer) - An integrated tool for analyzing your code, detecting bad practices, and catching bugs before you run your program. You can also configure it to only catch [PEP 8](https://www.python.org/dev/peps/pep-0008/) violations.
+
+[Debugging](https://docs.spyder-ide.org/current/panes/debugging.html) - This tool will help you debug your code, it allows you to set a `breakpoint` on a line. The debugging tool will then let you go through your python code and quickly notice where it goes wrong.
+
+[Variable Explorer](https://docs.spyder-ide.org/current/panes/variableexplorer.html) - This tool is really useful in combination with the debugger. It allows you to view all the active variables in your code and it even allows editing of the values inside those variables.
+
+---
+
+### Sublime text
+
+A text editor for coding, made by *Sublime HQ Pty Ltd*. It is similar to [VS Code](#visual-studio-code) in a lot of regards. Sublime text is almost fully customizable. It also has a built-in auto-complete engine and comes with Python syntax highlighting out-of-the-box.
+
+#### Notable extensions
+
+Sublime Text comes with a lot of tools already, but some of these tools could make your python development easier. Make sure you have *package control* installed, go to `Tools` >> `Install Package Control...`. To install packages open your command pallet and type in `Package Control: Install Package`, here you can enter the name of the packages you'd like to install.
+
+[Terminus](https://packagecontrol.io/packages/Terminus) - A terminal for your sublime. It is fully featured and support for Windows operating systems. This package is almost a must-have for you to be able to run `pytest` scripts.
+
+[SublimeLinter](https://packagecontrol.io/packages/SublimeLinter) - A linter for sublime. This tool automatically notifies you of bad-practice python code. Useful for when you have it set up to use PEP8.
+
+### JupyterLab
+
+Jupyter lab is an in-browser code editor that runs on your machine. You install it using a `pip` command in your *environment* and then automatically uses that environment. Jupyter is most well-known for being able to create coding notebooks. It is commonly used among data-scientists.
+
+#### Notable extensions
+
+The built-in tools for Jupyter Lab are good-enough for almost all exercises on *Exercism*. Most extensions made for Jupyter are made with data-science in mind. Explore extensions at your own pleasure.
```

Comment:
```suggestion
The built-in tools in Jupyter Lab are enough for the exercises on *Exercism*. Most extensions made for Jupyter are made with data-science in mind. Explore extensions at your own pleasure.
```

Reply:
"Enough", I am unsure if that is proper English. You say "Two days a week is enough" when you're talking about quantity, but do you say "The tool is enough for the job"? I am genuinely asking.

Reply:
I typically put a qualifier for the second case, but not a hyphenate:  
"Is the tool good enough for the job?".  
or I might re-phrase and go with  "Can the tool handle the job?" 
or "Does the tool have the capabilities to handle the job?".

Slang-y-er forms of it might be:
"Is the tool up to the task?" 
or "Can it handle the task?".  

"Is it enough for the task." is also valid, but can feel ambiguous or  abrupt without the qualifier.  

And -- as a native speaker of AmEn, I would defer to what an international speaker feels is _less_  ambiguous without being too formal, or too UK - feeling.

Reply:
and now that I really look at it, I might go with:

The built-in tools for Jupyter Lab are sufficient for almost all exercises on *Exercism*. Most extensions to Jupyter focus more on data-science applications and not general-purpose programming. Explore extensions at your own pleasure.

Reply:
> And -- as a native speaker of AmEn, I would defer to what an international speaker feels is less ambiguous without being too formal, or too UK - feeling.
Yep, I speak UK English, so... thank you!

I like this, thanks @BethanyG:
```suggestion
The built-in tools for Jupyter Lab are sufficient for almost all exercises on Exercism. Most extensions to Jupyter focus more on data-science applications and not general-purpose programming. Explore extensions at your own pleasure.
```